### PR TITLE
Deleted unused classes from site.scss

### PR DESCRIPTION
**Pull request by patfire12345**
full_width selector doesn't seemed to be used anywhere except for 1 place, where it was commented out.

### CODE REVIEW

**Review by ktw1016 — APPROVED** [review]
👍

File: client/src/common_css/site.scss [hunk 1]
```diff
@@ -144,10 +135,6 @@ strong,
   font-weight: $mediumFontWeight;
 }
 
-.light-weight {
-  font-weight: $lightFontWeight;
-}
```
Comment (Stephen-ONeil) [inline client/src/common_css/site.scss:149]: I'd have said to leave this for consistency with our other font weight utility classes.

File: client/src/panels/panel_declarations/finances/welcome_mat/welcome_mat.js [hunk 2]
```diff
@@ -477,12 +477,6 @@ const WelcomeMat = (props) => {
       />
     );
   } else if (type === "hist_estimates") {
-    //active, non-DP org, CR or program
-    //has no FTEs
-
-    //full-width,
-    //5 yrs ago, last year, this year, graph
-    //text about hist-diff
```
Comment (Stephen-ONeil) [inline client/src/panels/panel_declarations/finances/welcome_mat/welcome_mat.js:485]: This wasn't a comment about the `full-width` class name, this is just a generic description of the welcome mat's `type === "hist_estimates"` case. Not the kind of documentation I'd normally want in comments (should be clearer in the code/naming of things, would have to double check that these comments are still up to date with reality). The welcome mat is a huge mess though, so any additional help from comments (until it's cleaned up) is good to have.

**Review by Stephen-ONeil — COMMENTED** [review]
Post-merge review, oops. I'm going to hotfix my requested changes on master, just leaving my comments here to explain why.

**Pull request merged**